notebooks/scarlet-measure.ipynb

```diff
@@ -0,0 +1,2888 @@
+{
```

Comment:
<p>For the last plot, you might want to increase <code>figsize</code>so that values are readable? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSSTDESC/BlendingToolKit/pull/151/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Probably, this is bound to change with the plotting PR anyway.</p>